### DISCUSSION TITLE
Script for converting notes from Simplenote to Zim

### DISCUSSION THREAD

**Discussion by bruvellu** [zim-desktop-wiki/zim-desktop-wiki#2474] [Show and tell]
Hey all,

I made a simple script to convert notes from [Simplenote](https://www.simplenote.com/) to Zim. I used Simplenote for fleeting notes on my mobile and, from time to time, manually copied important notes to my main Zim notebook. But now that I have Markor/Syncthing set on my phone (works great), I needed something to batch convert my Simplenotes notes to Zim.

The script is available at: https://github.com/bruvellu/simplenote-to-zim

I hope it's useful for someone. Don't hesitate to open issues in the repository if you have any suggestions or encountered a problem.

Cheers,
Bruno

**Comment by introt** [comment]
Hi Bruno, thanks for releasing the script! Please add to the Zim Wiki -wiki for discoverability: https://github.com/jaap-karssenberg/zim-wiki/wiki/Conversion-scripts

**Reply by bruvellu** [reply]
Done!
